components/x-teaser/__fixtures__/opinion.json

```diff
@@ -27,5 +27,11 @@
   "indicators": {
     "isOpinion": true,
     "isColumn": true
-  }
+  },
+  "status": "",
```

Comment:
where these properties not required before?

Reply:
no, these are required now so that the dropdowns I added for the controls work

<img width="643" alt="Screenshot 2020-12-01 at 08 51 07" src="https://user-images.githubusercontent.com/20328348/100717747-62388a80-33b2-11eb-9b3b-b401006444af.png">